Meta/CMake/libtextcodec_generators.cmake

```diff
@@ -1,17 +0,0 @@
-function (generate_encoding_indexes)
-    set(LIBTEXTCODEC_INPUT_FOLDER "${CMAKE_CURRENT_SOURCE_DIR}")
-
-    # indexes.json can be found at https://encoding.spec.whatwg.org/indexes.json
-    invoke_generator(
```

Comment:
I think this patch would be a lot smaller if you kept the bespoke cmake file and just updated it to call the invoke_py_generator func, given the current state of the PR.